api/examples/python/README.md

```diff
@@ -8,16 +8,13 @@ notifications.
 First, install the dependencies.
 
 ```
-sudo /usr/bin/python -m pip install websocket-client
-sudo /usr/bin/python -m pip install protobuf
-
+mkvirtualenv iterm2api
```

Comment:
What's mkvirtualenv? I don't have it by default on my system. I agree sudo is awful, but I don't want to add even more dependencies on third party packages.